test-case/check-alsabat.sh

```diff
@@ -95,6 +96,20 @@ function __upload_wav_file
     done
 }
 
+function check_wav_file_snr
+{
+    for file in /tmp/bat.wav.*
+    do
+	if test -s "$file"; then
+	    dlogi "Checking wav file $file"
+	    cd "$TESTDIR"/tools
+	    octave --silent --no-gui --eval "check_wav_file_snr('$file', $frequency, '$LOG_ROOT/')" || {
```

Comment:
There should be ; in the end of octave command to avoid see "passed = 1" in test log